gems/pending/blackbox/VmBlackBox.rb

```diff
@@ -88,7 +88,7 @@ def deleteLocalDataDir(options={})
     end
 
     def writeData(filename, data)
-      Dir.mkdir(@localDataDir, 755) unless File.exist?(@localDataDir)

```

Comment:
Thanks, I wasn't sure if it was code that was executed or not.  Thanks @roliveri